WordPress/src/main/res/xml/locales_config.xml

```diff
@@ -0,0 +1,97 @@
+    <locale android:name="kmr"/> <!-- Present in available_languages, -->
+<!--    <locale android:name="kn"/> &lt;!&ndash; Kannada &ndash;&gt;-->
+    <locale android:name="ko"/> <!-- Korean -->
+<!--    <locale android:name="ky"/> &lt;!&ndash; Kyrgyz &ndash;&gt;-->
+<!--    <locale android:name="lo"/> &lt;!&ndash; Lao &ndash;&gt;-->
+<!--    <locale android:name="lt"/> &lt;!&ndash; Lithuanian &ndash;&gt;-->
+<!--    <locale android:name="lv"/> &lt;!&ndash; Latvian &ndash;&gt;-->
+    <locale android:name="mk"/> <!-- Macedonian -->
+<!--    <locale android:name="ml"/> &lt;!&ndash; Malayalam &ndash;&gt;-->
+<!--    <locale android:name="mn"/> &lt;!&ndash; Mongolian &ndash;&gt;-->
+<!--    <locale android:name="mr"/> &lt;!&ndash; Marathi &ndash;&gt;-->
+    <locale android:name="ms"/> <!-- Malay -->
+<!--    <locale android:name="my"/> &lt;!&ndash; Burmese &ndash;&gt;-->
+<!--    <locale android:name="my-MM"/> &lt;!&ndash; Burmese (Myanmar) &ndash;&gt;-->
+    <locale android:name="nb"/> <!-- Norwegian -->
+<!--    <locale android:name="ne"/> &lt;!&ndash; Nepali &ndash;&gt;-->
+    <locale android:name="nl"/> <!-- Dutch -->
+<!--    <locale android:name="or"/> &lt;!&ndash; Odia &ndash;&gt;-->
+<!--    <locale android:name="pa"/> &lt;!&ndash; Punjabi &ndash;&gt;-->
+    <locale android:name="pl"/> <!-- Polish -->
+    <locale android:name="pt-BR"/> <!-- Portuguese (Brazil) -->
+<!--    <locale android:name="pt-PT"/> &lt;!&ndash; Portuguese (Portugal) &ndash;&gt;-->
+    <locale android:name="ro"/> <!-- Romanian -->
+    <locale android:name="ru"/> <!-- Russian -->
+<!--    <locale android:name="si"/> &lt;!&ndash; Sinhala &ndash;&gt;-->
+    <locale android:name="sk"/> <!-- Slovak -->
+<!--    <locale android:name="sl"/> &lt;!&ndash; Slovenian &ndash;&gt;-->
+    <locale android:name="sq"/> <!-- Albanian -->
+    <locale android:name="sr"/> <!-- Serbian (Cyrillic) -->
+<!--    <locale android:name="sr-Latn"/> &lt;!&ndash; Serbian (Latin) &ndash;&gt;-->
+    <locale android:name="sv"/> <!-- Swedish -->
+<!--    <locale android:name="sw"/> &lt;!&ndash; Swahili &ndash;&gt;-->
+<!--    <locale android:name="ta"/> &lt;!&ndash; Tamil &ndash;&gt;-->
+<!--    <locale android:name="te"/> &lt;!&ndash; Telugu &ndash;&gt;-->
+    <locale android:name="th"/> <!-- Thai -->
+<!--    <locale android:name="tl"/> &lt;!&ndash; Filipino &ndash;&gt;-->
+    <locale android:name="tr"/> <!-- Turkish -->
+<!--    <locale android:name="uk"/> &lt;!&ndash; Ukrainian &ndash;&gt;-->
+<!--    <locale android:name="ur"/> &lt;!&ndash; Urdu &ndash;&gt;-->
+<!--    <locale android:name="uz"/> &lt;!&ndash; Uzbek &ndash;&gt;-->
+    <locale android:name="vi"/> <!-- Vietnamese -->
+    <locale android:name="zh-CN"/> <!-- Chinese (Simplified) -->
+    <locale android:name="zh-HK"/> <!-- Chinese (Hong Kong) -->
+    <locale android:name="zh-TW"/> <!-- Chinese (Traditional) -->
+<!--    <locale android:name="zu"/> &lt;!&ndash; Zulu &ndash;&gt;-->
+</locale-config>
+
```

Comment:
We'll want to auto generate this app using our release-toolkit when we download the list of locales from GlotPress. 

Basically replicate here what we already do for `available_languages.xml` (which we'll in turn be able to stop generating when we'll get rid of our custom in-app switcher once we fully migrate to this new Android 12 built-in per-app switcher)

Otherwise this list can easily go out of sync when we'll revisit the list of locales we include in the app [internal ref: pdnsEh-ev-p2]